sources/code/main/modules/extensions.ts

```diff
@@ -19,20 +19,22 @@ async function fetchOrRead(file:string, signal?:AbortSignal) {
  * 
  * **Experimental** – it is unknown if that would work properly for all themes.
  */
-async function parseImports(cssString: string, importCalls:string[], maxTries=5):Promise<string> {
+async function parseImports(cssString: string, importCalls: string[], maxTries=5):Promise<string> {
   const anyImport = /^@import .+?$/gm;
   if(!anyImport.test(cssString)) return cssString;
   const promises:Promise<string>[] = [];
+  const newImports = importCalls.slice();
   cssString.match(anyImport)?.forEach(singleImport => {
     const matches = /^@import (?:(?:url\()?["']?([^"';)]*)["']?)\)?;?/m.exec(singleImport);
     if(matches?.[0] === undefined || matches[1] === undefined) return;
-    const file = matches[1];
+    const lastUrl = importCalls[importCalls.length - 1] ?? "";
+    const file = resolve(lastUrl, matches[1]);
```

Comment:
I guess you might remove the `lastUrl` as well and just set it all in `file`, if you need a concise way to get the last element, `importCalls.at(-1)??""` feels like a good way (even [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#description) seems to mention this!).

Reply:
Done